lib/ui/page/home/page/chat/controller.dart

```diff
@@ -1184,6 +1185,77 @@ class ChatController extends GetxController {
     }
   }
 
+  /// Downloads the provided [ImageAttachment] or [FileAttachment].
+  Future<void> downloadMedia(Attachment attachment, {String? to}) async {
```

Comment:
Тут тоже можно:

```suggestion
  /// Downloads the provided [attachment].
  Future<void> downloadMedia(Attachment attachment, {String? to}) async {
```

Кстати, а если аттачмент `is LocalAttachment`, то вообще эти методы могут в теории вызваться? Если да, то стоит что-то сделать, чтобы нет, т.к. тут скачивание по урлам делается, могут быть баги.

Reply:
Хорошо, добавлю обработку такого случая.